NBrowse/src/Reflection/Mono/CecilType.cs

```diff
@@ -46,6 +46,17 @@ internal class CecilType : Type
 			this.definition?.Interfaces.Select(i => new CecilType(i.InterfaceType, this.project)) ??
 			Array.Empty<CecilType>();
 
+		public override IEnumerable<Type> GenericArguments
```

Comment:
I'm nitpicking here, but to keep consistency with the "Parameters" property do you think this one could be named "Arguments" instead? (another option would be to rename "Parameters" to "GenericParameters", but would affect Method class too).

Reply:
Sure thing, you're right.